dockerfiles/db/Dockerfile

```diff
@@ -1,25 +1,36 @@
 FROM postgres:13 as build
 
-RUN apt-get update
-RUN apt-get install build-essential git cmake curl -y
-RUN apt-get install postgresql-server-dev-13 -y
-RUN apt-get install python2 -y
+RUN \
+  apt-get update && \
+  apt-get install -y --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    cmake \
+    git \
+    postgresql-server-dev-13 \
+    python2 \
+    && \
+  rm -rf "/var/lib/apt/lists/*"
 
-RUN git clone https://github.com/graphql/libgraphqlparser.git \
-    && cd libgraphqlparser \
-    && cmake . \
-    && make install
+RUN \
+  git clone https://github.com/graphql/libgraphqlparser.git && \
```

Comment:
(optional) this is currently unpinned. if you're up for it, a good change would be:
- use the latest release `0.7.0` instead of master
- remove the `git`

Reply:
> remove the git

Do you mean the `.git` at the end of the URL?

Reply:
sorry, what i wrote made 0 sense.

I was trying to suggest an additional change to reduce the download size/time for libgraphqlparser.

Initially I thought this would be the best option
```
wget https://github.com/graphql/libgraphqlparser/archive/refs/tags/0.7.0.tar.gz
...
```
which would remove the need to install git. But on second thought, this would probably be better
```
git clone -b '0.7.0' --depth 1 git@github.com:graphql/libgraphqlparser.git
```

Reply:
That would require `wget` and `curl` to be on the image, I haven't checked if they are. This is more of a preference because the final image size is not impacted, and in my opinion it doesn't affect too much the build layers per see.

My changes ended up matching what you wanted, but the `--depth 1` which I am adding now.